ods_ci/tests/Tests/1300__model_registry/1301_model_registry_model_serving.robot

```diff
@@ -76,3 +71,14 @@ Verify Model Registry Integration With Secured-DB
     Maybe Wait For Dashboard Loading Spinner Page
     SeleniumLibrary.Page Should Contain Element    xpath:${MR_VERSION_TABLE_XPATH}/tbody/tr/td[@data-label="Version name"]//a[.="${MR_REGISTERED_MODEL_VERSION}"]
     SeleniumLibrary.Page Should Contain Element    xpath:${MR_VERSION_TABLE_XPATH}/tbody/tr/td[@data-label="Author" and .="${MR_REGISTERED_MODEL_AUTHOR}"]
+    SeleniumLibrary.Close All Browsers
+
+Verify Unallowed User Cannot See Model Registry From The Dashboard
+    [Documentation]    Negative path test for dashboard RBAC on the Model Registry. User not part of the group that is
+    ...    allowed to use a Model Registry instace should not be able to see it from the dashboard.
+    [Tags]    Smoke    MRMS1301    ModelRegistry
+    Depends On Test    Verify Model Registry Integration With Secured-DB
+    Launch Dashboard    ${TEST_USER_3.USERNAME}    ${TEST_USER_3.PASSWORD}    ${TEST_USER_3.AUTH_TYPE}
+    ...    ${ODH_DASHBOARD_URL}    ${BROWSER.NAME}    ${BROWSER.OPTIONS}
+    Open Model Registry Dashboard Page    allowed_user=${FALSE}
+    SeleniumLibrary.Page Should Contain    Request access to model registries
```

Comment:
isn't this already checked here https://github.com/red-hat-data-services/ods-ci/pull/2039/files#diff-d120a1bd56f9d1fb5afc47239d24ae89b2ae6da94f25d270134f304a412bea40R313

Reply:
It is, but I wanted to surface it as an explicit step in this TC just for clarity when going through the logs

Reply:
I think it makes more sense to have it defined this way because if the user is allowed we need to refresh the page in case the permissions have not been reconciled yet, and so the keyword will still need to be passed some variable to define if the user is allowed or not. 
Once you have that information in the keyword what makes the most sense IMHO is to have that logic defined in the keyword